package.json

```diff
@@ -34,17 +33,21 @@
   },
   "devDependencies": {
     "auto-changelog": "^1.4.6",
-    "ava": "^0.25.0",
+    "ava": "1.0.0-beta.4",
```

Comment:
Sneaking in more changes again hehe.. :-)

Reply:
Rebase artifact. I'll fix. 😄